pytrello2/board.py

```diff
@@ -35,10 +35,17 @@ def get_all_boards(self):
 
     def create_board(self, name, desc):
         """
-        Returns a Card object with the given ID.
+        Returns a Board object with the given ID.
         """
         data = {}
         data["name"] = name
         data["desc"] = desc
         board_data = self.client.post("boards/", data)
         return Board(board_data)
+
+    def delete_board(self, board_id):
+        """
+        Returns a Board object with the given ID.
```

Comment:
The doc string should explain what the function is about. Could we say something like deletes a board?

WDYT?

Reply:
Agreed, will change it.